Sources/CorePayments/Networking/Enums/Environment.swift

```diff
@@ -32,3 +32,4 @@ public enum Environment {
         }
     }
 }
+// swiftlint:enable force_unwrapping
```

Comment:
the new version of swiftlint requires it. That is why most of the iOS prs where failing, because we didnt re enable whatever rule we disabled before. Unless we do a `disable:next`, it asks to enable again.